securedrop_client/gui/widgets.py

```diff
@@ -790,35 +795,35 @@ def __init__(self, source: Source):
         summary_layout.setSpacing(0)
         self.name = QLabel()
         self.name.setObjectName('source_name')
-        self.preview = QLabel('')
+        self.preview = QLabel()
         self.preview.setObjectName('preview')
+        self.preview.setFixedSize(QSize(365, 60))
```

Comment:
looking at zeplin I think we want to make this 40px in height (the total height of this source item in zeplin is 85 px, this change as is increases the size of the source list items significantly):  

<img width="582" alt="Screen Shot 2019-08-02 at 12 46 04 PM" src="https://user-images.githubusercontent.com/7832803/62385743-deebf800-b523-11e9-89c4-81a3784b6204.png">

Reply:
Part of the prob @redshiftzero is that QT won't accept the standard CSS syntax for `line-height,` which is what will give the preview block that height. Allie's been a mensch trying to research how to overcome this, but it's cut into her time to tackle other sprint priorities, and Erik asked her to punt on that for now. Might you have any ideas how to set line-height in QT?

Reply:
so i think the line-height is a separate issue from what i'm flagging, in this diff we are hard coding the size of the preview block, e.g. this is the height set at 60px (what's currently in this PR):

<img width="469" alt="Screen Shot 2019-08-02 at 1 22 24 PM" src="https://user-images.githubusercontent.com/7832803/62387471-b0245080-b528-11e9-86c6-4acd3385dad8.png">

and here's with it set at 40px which is what is in the spec:

<img width="477" alt="Screen Shot 2019-08-02 at 1 22 02 PM" src="https://user-images.githubusercontent.com/7832803/62387491-bca8a900-b528-11e9-8760-9006dc29d842.png">

Reply:
so regarding the line-height: indeed it's not supported via the subset of CSS that Qt supports, I just took a look at the Qt API to see if there's a way to set the line height not using CSS, and it looks like one option would be:

1. create [QTextEdit](https://doc.qt.io/qt-5/qtextedit.html) object in [read-only mode](https://doc.qt.io/qt-5/qtextedit.html#read-only-key-bindings)
2. that gives us access via the `document` property on `QTextEdit` to [QTextDocument](https://doc.qt.io/qt-5/qtextdocument.html) which holds formatted text
3. we can then control line height via [QTextBlockFormat.setLineHeight()](https://doc.qt.io/qt-5.9/qtextblockformat.html) - `QTextBlockFormat` is the object which lets us specify formatting for text in a `QTextDocument`.

(indeed this is a lot of effort for that line-height so I would defer for now until we fry these larger fish)

Reply:
Gotcha! Yes, line-height is a fish to fry later... fried fish, yum!

I'm looking at the two screenshots (thank you!) you posted, and as a non-developer am seeing too much margin-space (or padding outside) on the preview block. The Source name and timestamp should indeed be positioned more closely to the preview block, with the star horizontally aligning along the middle with the Source's name (it currently looks like they're top-aligning, vs middle-aligning, horizontally).

Reply:
@redshiftzero ah thx for pointing this out. i will update it to 40 since there will not be preview text yet and less space looks better and when we add the preview text we can expand this space to 60px if there are 3 lines as show in zeplin: https://app.zeplin.io/project/5c807ea562f734bd2756b243/screen/5d25217bd4b8fc98d7b13bab.

Reply:
sorry for the run-on sentence